tools/coreml/test/test_mxnet_converter.py

```diff
@@ -192,6 +192,15 @@ def test_tiny_tanh_activation_random_input(self):
         net = mx.sym.Activation(net, name='tanh1', act_type="tanh")
         self._test_mxnet_model(net, input_shape=input_shape, mode='random')
 
+    def test_tiny_prelu_leakyrelu_random_input(self):
+        np.random.seed(1988)
+        input_shape = (1, 10)
+        net = mx.sym.Variable('data')
+        net = mx.sym.FullyConnected(data=net, name='fc1', num_hidden=5)
+        gamma = mx.sym.Variable('gamma')
+        net = mx.sym.LeakyReLU(net, gamma=gamma, name='prelu1', act_type="prelu")
```

Comment:
While we are at this, do you think it'll be useful to add support for act_type="leaky" as well and not only for the advanced version of prelu? If you think you'd rather not do that in this change, then it is fine too but just wanted to throw it out there.

Reply:
Sure, I will add "leaky" and "elu".

Reply:
Done in da9f7cb